### PR TITLE
VIH-9933 Update cache to an empty array if a justice user does not exist

### PR DESCRIPTION
**Pull request by shaed-parkar**
### JIRA link (if applicable) ###

VIH-9933

### Change description ###

* Update cache to an empty array if a justice user does not exist
* Move away from midddleware to OnTokenValidated Event to avoid multiple async calls to Bookings API

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by phildebrah — APPROVED** [review]
seen

File: VideoWeb/VideoWeb/appsettings.Development.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
   "Logging": {
     "LogLevel": {
-      "Default": "Debug",
-      "System": "Debug",
-      "Microsoft": "Debug",
+      "Default": "Warning",
+      "System": "Warning",
+      "Microsoft": "Warning",
```
Comment (will-craig) [inline VideoWeb/VideoWeb/appsettings.Development.json:6]: Don't thing we need to commit changes to the development.json

File: VideoWeb/VideoWeb/Services/AppRoleService.cs [hunk 2]
```diff
@@ -49,14 +49,20 @@ public async Task<List<Claim>> GetClaimsForUserAsync(string uniqueId, string use
...
-            claims.Add(new Claim(ClaimTypes.GivenName, user.FirstName));
-            claims.Add(new Claim(ClaimTypes.Surname, user.Lastname));
-            claims.Add(new Claim(ClaimTypes.Name, user.FullName));
+            if (user == null)
+            {
+                claims = new List<Claim>();
+            }
+            else
+            {
+                claims = MapUserRoleToAppRole(user.UserRoleId);
```
Comment (will-craig) [inline VideoWeb/VideoWeb/Services/AppRoleService.cs:60]: userRoleId doesnt exist anymore, it's an array of role object, if you update the booking.client - i have a pr to fix this https://github.com/hmcts/vh-video-web/pull/2018/files but if it makes more sense, can do it in this change  and ill close my pr. Up to you

File: VideoWeb/VideoWeb/appsettings.Development.json (hunk 1 above)
Comment (shaed-parkar) [reply VideoWeb/VideoWeb/appsettings.Development.json:6]: that's alright. I thought we were going to lower the logs for dev too after the feedback from the las retro

File: VideoWeb/VideoWeb/Services/AppRoleService.cs (hunk 2 above)
Comment (shaed-parkar) [reply VideoWeb/VideoWeb/Services/AppRoleService.cs]: Thanks, I ended up copying the implementation from Admin Web

**Review by will-craig — APPROVED** [review]
Nice one

**Review by phildebrah — APPROVED** [review]
Re approved

**Pull request merged**